seqr/views/utils/export_utils.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import unicode_literals
+from builtins import str
```

Comment:
you should never need to import builtins, they should already be in scope

Reply:
The `builtins` is part of the future package. It makes type `str` to become unicode.